### DISCUSSION TITLE
troubles switching hash router with browser router

### DISCUSSION THREAD

**Discussion by exaucae** [karakum-team/kotlin-mui-showcase#20] [Q&A]
Not an issue per se. Would like to know why the navigation doesn't work when using browserRouter insead of hash router. Eg, https://karakum-team.github.io/kotlin-mui-showcase/material-ui%2Freact-slider would dyield
> Cannot GET /material-ui%2Freact-slider

**Comment by aerialist7** [comment]
Hi, @exaucae! 
I deploy my showcases on GitHub pages where my app is not a host app, so I can't use `BrowserRouter` there.
So, the answer - because of the environment :)
